tests/benchdnn/inputs/matmul/harness_matmul_decompression

```diff
@@ -178,32 +178,37 @@
 --wtag=any,ab,ba
 --dt=s8:s8:f16
 --attr-scales=src:common:0.5:f32+wei:per_oc:f16
+--attr-fpmath=f16:true
```

Comment:
This was designed to compute int8-int8. fpmath mode would force up conversion. Please keep both versions.
```suggestion
--attr-fpmath=,f16:true
```